pulp_service/pulp_service/app/authorization.py

```diff
@@ -0,0 +1,52 @@
+from contextvars import ContextVar
+from base64 import b64decode
+from binascii import Error as Base64DecodeError
+import json
+import jq
+from pulpcore.app import settings
+from pulp_service.app.models import DomainOrg
+from rest_framework.permissions import BasePermission
+import logging
+
+
+_logger = logging.getLogger(__name__)
+org_id_var = ContextVar('org_id')
+org_id_json_path = jq.compile(".identity.internal.org_id")
```

Comment:
Should we make this configurable to make it work with cert and console.dot service accounts? Do they use the same path to their `org_id`s?